pkg/controllers/readiness/unsupported_override.go

```diff
@@ -54,14 +55,16 @@ func isUnsupportedUnsafeAuthentication(spec *operatorv1.OperatorSpec) (bool, err
 	}
 }
 
-func getExpectedMinimumNumberOfMasters(spec *operatorv1.OperatorSpec) int {
+func getExpectedMinimumNumberOfMasters(spec *operatorv1.OperatorSpec, topologyMode configv1.TopologyMode) int {
 	allowAnyNumber, err := isUnsupportedUnsafeAuthentication(spec)
 	switch {
 	case err != nil:
 		utilruntime.HandleError(err)
 		return 3
```

Comment:
Shouldn't the topology mode override this default of 3 as well?

Reply:
fixed